### ISSUE TITLE
embedded navigation

### ISSUE THREAD

**Issue by fabiolin** [nextbillion-ai/nb-navigation-flutter#75]
does this package have embedded navigation turn by turn?

**Comment by zhengshenqiu** [comment]
Yes, If you want to use a embedded widget as the navigation view , you can use **NBNavigationView** widget  to lanch the navigation. An example provided here : https://github.com/nextbillion-ai/nb-navigation-flutter/blob/main/example/lib/launch_embedded_navigation_view.dart
